plugin.video.playonbrowser/addon.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.playonbrowser" name="PlayOn Browser" version="2.0.0" provider-name="Lunatixz">
+    <requires>
+        <import addon="xbmc.python" version="2.25.0"/>
+        <import addon="script.module.simplecache" version="1.0.0"/>
+        <import addon="script.module.xmltodict" version="0.9.0"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="default.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en_GB">Browse PlayOn from Kodi.</summary>
+        <description lang="en_GB">PlayOn is a streaming media brand and software suite that enables users to view and record videos from numerous online content providers.</description>
+        <disclaimer lang="en_GB">Requires Playon Server Software</disclaimer>
+        <platform>all</platform>
+        <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
+        <forum>http://forum.kodi.tv/showthread.php?tid=233136</forum>
+        <website>https://www.playon.tv</website>
+        <source>https://github.com/Lunatixz/XBMC_Addons/tree/master/plugin.video.playonbrowser</source>
+        <news>v.2.0.0[CR]- Refactored and optimized Code.[CR]- Added PlayLater Recording via Context Menu.</news>
+        <assets>
+            <icon>resources/images/icon.png</icon>
+            <fanart>resources/images/fanart.jpg</fanart>
+            <screenshot>resources/images/screenshot01.png</screenshot>
```

Comment:
I am not sure an addon can have more than 4 screenshots. This is probably too much for the server infrastructure (please keep them to a minimum, let's say 4). The screenshot work was primally intended for skins and scripts in which you skin your addon. Plugins only manipulate xbmc vfs so they all look the same. In fact that's one of the advantages of kodi itself...having a consistent interface.